crates/uv-git/src/git.rs

```diff
@@ -970,10 +458,6 @@ pub(crate) fn fetch(
         }
     };
 
-    maybe_gc_repo(repo)?;
-
-    clean_repo_temp_files(repo);
```

Comment:
Why remove these two?

Reply:
Both of those were linked to libgit2 issues. The git cli runs garbage collection automatically on `git fetch`, `git commit`, and other commands, so it shouldn't be a problem now.